packages/core/src/Stores/client-store.js

```diff
@@ -2130,7 +2130,10 @@ export default class ClientStore extends BaseStore {
 
             const redirect_url = sessionStorage.getItem('redirect_url');
 
-            if (redirect_url?.endsWith('/') && (isTestLink() || isProduction() || isLocal() || isStaging())) {
+            if (
+                (redirect_url?.endsWith('/') || redirect_url.endsWith(routes.bot)) &&
```

Comment:
```suggestion
                (redirect_url?.endsWith('/') || redirect_url.endsWith(routes.bot)) &&
```

I think the check for `redirect_url?.endsWith` should be removed, but left it here and instead added ` redirect_url.endsWith(routes.bot)` to it, incase removing it breaks something